rules/windows/lateral_movement_direct_outbound_smb_connection.toml

```diff
@@ -47,6 +46,10 @@ license = "Elastic License v2"
 name = "Direct Outbound SMB Connection"
 note = """## Triage and analysis
 
+### Performance
```

Comment:
its good to have a description of the performance impact reasoning, but for users who want to enable/disable certain rules via rules management GUI they will need a faster way to do so, tags maybe ?